backend/compact-connect/lambdas/provider-data-v1/tests/function/test_data_model/test_provider_transformations.py

```diff
@@ -5,7 +5,7 @@
 from boto3.dynamodb.conditions import Key
 from moto import mock_aws
 
-from tests.function import TstFunction
+from .. import TstFunction
```

Comment:
If we switch to relative imports to avoid test package name conflicts, we should probably do it everywhere.